src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -219,6 +219,13 @@ describe('sidebar.components.sidebar-content', function() {
 
       assert.isFalse(store.getState().directLinkedGroupFetchFailed);
     });
+
+    it("clears the direct linked id's from the settings", () => {
```

Comment:
The test description here needs updating to reflect the fact that the state was moved to the store.